src/config/axios.ts

```diff
@@ -0,0 +1,5 @@
+export const REACT_APP_GLIFIC_REGISTRATION_API =
+  process.env.REACT_APP_GLIFIC_API + '/v1/registration';
+export const REACT_APP_GLIFIC_AUTHENTICATION_API =
+  process.env.REACT_APP_GLIFIC_API + '/v1/registration/send_otp';
+export const REACT_APP_GLIFIC_CHAT_REDIRECT = 'http://localhost:3000/chat';
```

Comment:
let's update this too. Something like:
```suggestion
export const REACT_APP_GLIFIC_CHAT_REDIRECT = process.env.REACT_APP_GLIFIC_URL . '/chat';
```